graphql-server/src/branches/branch.model.ts

```diff
@@ -37,6 +43,15 @@ export class BranchList extends ListOffsetRes {
   list: Branch[];
 }
 
+@ObjectType()
+export class AheadOrBehind {
```

Comment:
Idea for a clearer name but it's more verbose
```suggestion
export class RemoteBranchDiffCounts {
```